tests/conftest.py

```diff
@@ -1,9 +1,22 @@
 from __future__ import annotations
 
+import os
 from pathlib import Path
+from typing import TYPE_CHECKING
+from unittest import mock
```

Comment:
Use pytest-mock instead like this repo does everywhere else.

Reply:
@gaborbernat I was not able to make it work with either pytest-mock or pytest monkeypath... tried for a couple hours. Funny aspect is that tox itself is using it at https://github.com/tox-dev/tox/blob/e3e77a6f711f0c333aea10eb2bc8794c6215c637/tests/conftest.py#L130

In fact despite having pytest-mock listed as a test dependency, this project does not seem to have any use of `mocker` object.

Note: I supposed one of the reasons why this might not work as desired is because it the `tox_project: ToxProjectCreator` fixture might not work well it (not sure).

Any suggestions? -- for the moment I removed my tox user config to be able to run the tests on tox-uv.